tests/performance-test/performance-test.sh

```diff
@@ -158,37 +237,41 @@ else
     usage
 fi
 
-get_sources
-make_grafana_datasources
+# Test groundwork
+check_resources
 make_service_monitors
+delete_es_events
 post_dashboards
 
+SUCCESS_RATIO=0.00
+post_ratio_to_es
+
+# Test procedure
 STAGE="TARGET"
 while true; do
     case $STAGE in
         "TARGET")
-            TARGETS=$(curl "http://$(oc get route prometheus -o jsonpath='{.spec.host}')/api/v1/targets")
+            TARGETS=$(curl --silent "http://$(oc get route prometheus -o jsonpath='{.spec.host}')/api/v1/targets")
             QDR=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"qdr-white"')
             PROM=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"prometheus-operated"')
 
             if [ -z "$QDR" ] && [ -z "$PROM" ]; then
                 printf "%s" "Waiting for new targets to be recognized by Prometheus Operator"; ellipse
-                printf "\r"
                 sleep 10
             else
                 echo "Found new target endpoints"
+                make_qdr_edge_router
```

Comment:
:question: I'm interested to know why this moved?

Reply:
I believe it may be wrong in the master branch - if the edge routers are not established already, then the loop will continuously tell oc to build the monitors, even if oc is already trying to